src/Jaeger/SpanConverter.php

```diff
@@ -125,15 +125,35 @@ private static function convertTimestamp(\DateTimeInterface $dateTime)
         return (int)((float) $dateTime->format('U.u') * 1000 * 1000);
     }
 
+    const BIG_MATH_BC = 'bc';
+    const BIG_MATH_GMP = 'gmp';
+
+    private static $preferredBigMathExt = 'bc';
+
+    public static function setPreferredBigMathExt($ext = self::BIG_MATH_BC)
+    {
+        self::$preferredBigMathExt = $ext;
+    }
+
     /**
      * Split the provided hexId into 2 64-bit integers (16 hex chars each).
      * Returns array of 2 int values.
      */
     private static function convertTraceId($hexId)
     {
+        $method = '';
+        if (self::$preferredBigMathExt === self::BIG_MATH_BC) {
```

Comment:
Can we turn this into an interface with 2 separate implementations and let us not have to do this check every time? We can also potentially detect the default implementation based on which extensions are loaded.